PennMobile/Dining/SwiftUI/Views/DiningViewHeader.swift

```diff
@@ -24,7 +25,8 @@ struct DiningViewHeader: View {
             }
             HStack {
                 DiningBalanceView(description: "Guest Swipes", image: Image(systemName: "person.2.fill"), balance: Double(diningVM.diningBalance.guestVisits), specifier: "%.0f")
-                AnalyticsCardView(text: "Coming Soon!")
+                AnalyticsCardView(text: "Analytics!")
```

Comment:
Maybe removing the exclamation mark for consistency 😅

Reply:
Sounds good!